presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoTableHandle.java

```diff
@@ -29,19 +30,22 @@
 {
     private final SchemaTableName schemaTableName;
     private final TupleDomain<ColumnHandle> constraint;
+    private final OptionalLong limit;
```

Comment:
Can we maintain it as `OptionalInt` as we won't pushdown  limit if is beyond `Integer` range.

Reply:
Updated to OptionalInt from OptionalLong.